src/pages/dashboard/dashboard.page.tsx

```diff
@@ -0,0 +1,73 @@
+import React from "react";
+import { Grid, Paper } from "@mui/material";
+import Chart from "react-apexcharts";
+import { AccountInfo } from "./components/account-info";
+import { Box } from "@mui/system";
+import { users } from "../../data/user";
+import { FundsTransfer } from "./components/funds-transfer";
+
+const Dashboard = () => {
+  return (
+    <Grid
+      container
+      direction="column"
+      justifyContent="center"
+      alignItems="stretch"
+      spacing={5}
+    >
+      <Grid item>
+        <AccountInfo />
+      </Grid>
+      <Grid item>
+        <FundsTransfer />
+      </Grid>
+      <Grid item>
+        <Paper elevation={3} variant="elevation">
+          <Box p={3}>
+            <Chart
+              height={500}
+              options={{
+                chart: {
+                  type: "line",
+                  height: 300,
+                  zoom: {
+                    enabled: false,
+                  },
+                },
+                dataLabels: {
+                  enabled: false,
+                },
+                stroke: {
+                  curve: "straight",
+                },
+                title: {
+                  text: "Transaction Graph (Last 15)",
+                  align: "left",
+                },
+                xaxis: {
+                  categories: users[0].transactions
```

Comment:
dont use users data directly, set a state and use it from there, like done in the previous components. Since we always fetch data from the server and store it in a state.

Reply:
Moved to a separate component folder with changes